### PR TITLE
Bootstrap kickoff

### PR DESCRIPTION
**Pull request by gabrielbori**
Created base project. Created example test.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,61 @@
...
+```
+
+For Android
+
+```
+yarn android.app
+```
+
+
+## Deployment
```
Comment (gaboxxz) [inline README.md:42]: delete

File: README.md [hunk 2]
```diff
@@ -1 +1,137 @@
-# appium-bootstrap-frontend
\ No newline at end of file
+# Appium Bootstrap
+
+This projects is the base to start automating mobile apps.
```
Comment (ffasitella) [inline README.md:3]: project :sweat_smile:

File: config/wdio.android.app.conf.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+const { config } = require('./wdio.shared.conf');
+
+config.cucumberOpts.tagExpression = `${config.cucumberOpts.tagExpression} and not @ios`;
+// ============
+// Capabilities
+// ============
+// For all capabilities please check
+// http://appium.io/docs/en/writing-running-appium/caps/#general-capabilities
+config.capabilities = [
```
Comment (ffasitella) [inline config/wdio.android.app.conf.js:9]: Leave an empty line between the comment and the start of this block

File: config/wdio.android.app.conf.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  {
+    platformName: 'Android',
+    maxInstances: 1,
+    'appium:deviceName': 'Android Emulator',
+    'appium:orientation': 'PORTRAIT',
+    'appium:automationName': 'UiAutomator2',
+    'appium:app': process.env.ANDROID_APP,
+    'appium:newCommandTimeout': 2400,
+    'appium:chromedriverDisableBuildCheck': true,
+
```
Comment (ffasitella) [inline config/wdio.android.app.conf.js:19]: Remove extra line

File: config/wdio.shared.conf.js [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+  //
+  // If you only want to run your tests until a specific amount of tests have failed use
+  // bail (default is 0 - don't bail, run all tests).
+  bail: 0,
+  //
+  // Set a base URL in order to shorten url command calls. If your `url` parameter starts
+  // with `/`, the base url gets prepended, not including the path portion of your baseUrl.
+  // If your `url` parameter starts without a scheme or `/` (like `some/path`), the base url
+  // gets prepended directly.
+  baseUrl: 'http://localhost',
```
Comment (ffasitella) [inline config/wdio.shared.conf.js:74]: Maybe I would put a comment before this line, suggesting to store this url in an env var.

File: features/home/home.steps.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+});
+
+When(/^i am in the home screen$/, () => {
+  HomeScreen.waitForIsShown();
+});
+
+Then(/^i should see a the text (.*)$/, text => {
+  const textElement = HomeScreen.getElementByText(text);
+  expect(textElement.isDisplayed()).toBe(true);
+});
```
Comment (ffasitella) [inline features/home/home.steps.js:15]: Add empty line

File: screens/Home.screen.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+import AppScreen from './app.screen';
+const SELECTORS = {
+  HOME_SCREEN: driver.isAndroid
+    ? '-android uiautomator: new UiSelector().description("Home-screen")'
+    : '~Home-screen',
+  TEXT: driver.isAndroid
+    ? '-android uiautomator: new UiSelector().text("{TEXT}")'
+    : '//XCUIElementTypeStaticText[@name="{TEXT}"]'
+};
```
Comment (ffasitella) [inline screens/Home.screen.js:9]: This code could be a little difficult to understand to someone new. Maybe put a comment with a short description about what it does?

**Review by ffasitella — CHANGES_REQUESTED** [review]
Some small comments. Nice first version!

File: README.md (hunk 2 above)
Comment (gaboxxz) [reply README.md:3]: cuak.

File: config/wdio.android.app.conf.js (hunk 3 above)
Comment (gaboxxz) [reply config/wdio.android.app.conf.js:9]: done

File: config/wdio.android.app.conf.js (hunk 4 above)
Comment (gaboxxz) [reply config/wdio.android.app.conf.js:19]: done

File: config/wdio.shared.conf.js (hunk 5 above)
Comment (gaboxxz) [reply config/wdio.shared.conf.js:74]: ok

File: features/home/home.steps.js (hunk 6 above)
Comment (gaboxxz) [reply features/home/home.steps.js:15]: ok

File: screens/Home.screen.js (hunk 7 above)
Comment (gaboxxz) [reply screens/Home.screen.js]: done

**Review by ffasitella — DISMISSED** [review]
:clap:

**Review by ffasitella — APPROVED** [review]

**Pull request merged**